### ISSUE TITLE
null error 

### ISSUE THREAD

**Issue by nathans212** [SamJakob/SpiGUI#2]
I get this error when clicking the arrows

```java.lang.NullPointerException: null
        at spigui.SGMenu.nextPage(SGMenu.java:444) ~[?:?]
        at spigui.SpiGUI.lambda$new$2(SpiGUI.java:96) ~[?:?]
        at spigui.menu.SGMenuListener.onInventoryClick(SGMenuListener.java:64) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor846.execute(Unknown Source) ~[?:?]```

Server version
 This server is running Paper version git-Paper-385 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)

**Comment by stijnb1234** [comment]
Same for me

```
java.lang.NullPointerException: null
        at nl.sbdeveloper.showapi.helpers.guihelper.SGMenu.nextPage(SGMenu.java:443) ~[?:?]
        at nl.sbdeveloper.showapi.helpers.guihelper.SpiGUI.lambda$null$2(SpiGUI.java:94) ~[?:?]
        at nl.sbdeveloper.showapi.helpers.guihelper.menu.SGMenuListener.onInventoryClick(SGMenuListener.java:64) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor579.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.4.jar:git-Paper-267]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.4.jar:git-Paper-267]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.4.jar:git-Paper-267]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2526) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:32) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1136) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1129) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1203) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1000) ~[patched_1.16.4.jar:git-Paper-267]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.4.jar:git-Paper-267]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_262]
```

**Comment by stijnb1234** [comment]
Fixed in #4

**Comment by stijnb1234** [comment]
Please close
